docs/developer/deploy/upgrade.rst

```diff
@@ -0,0 +1,57 @@
+Upgrade a Platform with the latest dev build
+============================================
+
+Prerequisites
+-------------
+- A Platform already installed with all pod up & running
+- A new metalk8s.iso image with a higher version
+
+Procedure
+---------
+
+If upgrading from a lower patch, minor or major version just follow the
+`standard upgrade procedure <https://metal-k8s.readthedocs.io/en/latest/operation/upgrade.html>`_.
```

Comment:
This page lives in the same documentation, so we should use relative references instead of plain HTTP hyperlinks:

```suggestion
:doc:`standard upgrade procedure </operation/upgrade>`.
```

Reply:
done